cmd/src/cli/js-tests-scaffold/index.js

```diff
@@ -1,21 +1,38 @@
 // This test file uses the tape testing framework.
 // To learn more, go here: https://github.com/substack/tape
 const test = require('tape');
-const Container = require('@holochain/holochain-nodejs');
 
-// instantiate an app from the DNA JSON bundle
-const app = Container.instanceFromNameAndDna("app", "dist/bundle.json")
+const { Config, Container } = require("@holochain/holochain-nodejs")
 
-// activate the new instance
-app.start()
+//const dnaPath = "./dist/app_spec.hcpkg"
+const dnaPath = "./dist/bundle.json"
+
+// closure to keep config-only stuff out of test scope
+const container = (() => {
+    const agentAlice = Config.agent("alice")
+
+    const dna = Config.dna(dnaPath)
+
+    const instanceAlice = Config.instance(agentAlice, dna)
+
+    const containerConfig = Config.container(instanceAlice)
```

Comment:
#827 introduces one more small change, where `Config.container` accepts a single array rather than multiple arguments, because in later iterations we'll likely need to pass config as well. I actually updated this file over in that branch too, we should probably use the one here so whoever merges this feel free to toss out the other one. (It may be me who deals with that conflict, so note to self)

```suggestion
    const containerConfig = Config.container([instanceAlice])
```

Reply:
updated